homeassistant/components/plex/media_player.py

```diff
@@ -491,14 +491,22 @@ def play_media(self, media_type, media_id, **kwargs):
                 ) from err
         else:
             shuffle = src.pop("shuffle", 0)
+            continuous = src.pop("continuous", 0)
+            continuation = src.pop("_continue", None)
```

Comment:
Is there a need to prefix the service call kwarg with "_"?

Reply:
We shouldn't use leading `_` as they will be perceived as protected

Reply:
I like the sound of `resume_mode`; would the word "resume" still be appropriate in the context of my [last comment](https://github.com/home-assistant/core/pull/51180#issuecomment-907706650)?

Just a consideration for potential future plans, but if this naming ambiguity is not a concern - I'm all in favor of the `resume_mode` option. If the ambiguity is concerning, I'm open to any alternative suggestions :)

Reply:
Taking guidance from later [comments](https://github.com/home-assistant/core/pull/51180#issuecomment-909670225), `_continue` was renamed to `episode_resume_query`